src/app/dashboard/actions.ts

```diff
@@ -14,6 +17,9 @@ export async function createBudget(
     user_id,
     event_name,
     event_description,
+    event_location,
+    // event_date,
```

Comment:
the budget's `event_date` should be set based on the value passed to the function in line 10